View/multiworldClient.py

```diff
@@ -1,29 +1,118 @@
 import asyncio
 
 from PySide6.QtWidgets import *
+from PySide6.QtCore import QThread, QObject, Signal, Slot
 
 from View.uiMultiworldClient import uiMultiworldClient
 from Model.serverConfig import ServerConfig
 from Model.setUpDto import SetUpDto
 import Client.clientCommunication as clientFunctions
 
+from Model.config import Config
+
+
+class JoinServerWorker(QObject):
+    
+    message = Signal(str) # to communicate with parent (gui) thread
+
+    def __init__(self) -> None:
+        super(JoinServerWorker, self).__init__()
+        self.config = Config.get_config()
+
+
+    def run(self) -> None:
+        print("Setting Fields")
+        server_config = ServerConfig(self.config._ServerAddress, self.config._Port,
+        self.config._World_id, 'admin', 'adminPass')
+        set_up_dto = SetUpDto(1, self.config._Game_Room, None, False) # 1 = world amount, just 1 for testing to match with number assigned to input box
```

Comment:
Although the world count isn't used yet in the server I'd prefer if you just linked it anyways. Since the current setup is already in your brain it would be easiest to do it now.